ecosystem/sep-0010.md

```diff
@@ -97,12 +99,20 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * source account set to server's signing account
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
-  * operations: `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-    * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
-    * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
+  * operations:
+    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
+      * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
+      * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
+    * `manage_data(source: server_account, key: 'server_hostname', value: server_hostname)`
```

Comment:
So @leighmcculloch as it is currently, the server hostname op must be 2nd. If your intention is to treat the operations more like an unordered set, we need to be a lot more clear on that.

I can imagine most developers accessing these ops will do so with explicit indexes. Currently, I'm sure many do something like:
```
challenge.operations[0]
```

Reply:
Yup, anything after the first op is essentially an unordered set of key-value pairs.
https://github.com/stellar/stellar-protocol/blob/0e537c56ff99edc47af175783a17fc32b7fd0da6/ecosystem/sep-0010.md#L112-L114

Reply:
Ok, then I don't think we should have the 2nd manage data operation listed in the `transaction` section explicitly as the 2nd op.

Reply:
I've made the unordered and ordered nature of the ops explicit in two places, how does that look? e0fbb13 ff7a51a